packages/assets-controllers/CHANGELOG.md

```diff
@@ -6,6 +6,14 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [9.0.0]
+### Added
+- **BREAKING**: Add multi account balances fetch flag support ([#1146](https://github.com/MetaMask/core.git/pull/1146))
```

Comment:
Hmm. It appears that all of these new changelog entries are using `https://github.com/MetaMask/core.git` for the repo URL, but it actually needs to be `https://github.com/MetaMask/core`. I'll have to look into why this is happening. In the meantime I'll add some suggestions for replacing these. This should get CI to pass the "validate changelog" step.

Reply:
Also maybe we flesh this out a bit based on the changelog entries listed in this PR:

``` suggestion
- **BREAKING**: Add required options `getSelectedAddress` and `getMultiAccountBalancesEnabled` to AccountTrackerController constructor and make use of them when refreshing account balances ([#1146](https://github.com/MetaMask/core/pull/1146))
  - Previously, the controller would refresh all account balances, but these options can be used to only refresh the currently selected account